plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/BaseTestElasticsearchConnectorTest.java

```diff
@@ -39,22 +41,22 @@
 import static io.trino.spi.type.BigintType.BIGINT;
 import static io.trino.spi.type.VarcharType.VARCHAR;
 import static io.trino.testing.MaterializedResult.resultBuilder;
+import static io.trino.testing.QueryAssertions.assertContains;
 import static io.trino.testing.assertions.Assert.assertEquals;
 import static java.lang.String.format;
 import static java.nio.charset.StandardCharsets.UTF_8;
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.testng.Assert.assertFalse;
 import static org.testng.Assert.assertTrue;
 
-public abstract class BaseElasticsearchSmokeTest
-        // TODO extend BaseConnectorTest
-        extends AbstractTestIntegrationSmokeTest
+public abstract class BaseTestElasticsearchConnectorTest
+        extends BaseConnectorTest
 {
     private final String image;
     private ElasticsearchServer elasticsearch;
     protected RestHighLevelClient client;
 
-    BaseElasticsearchSmokeTest(String image)
+    BaseTestElasticsearchConnectorTest(String image)
```

Comment:
nit: existing convention is `BaseElasticsearchConnectorTest` for the abstract base classes. See `BaseSqlServerConnectorTest`.

Reply:
@hashhar  I've followed here the requirements from https://github.com/trinodb/trino/issues/7109 . I find though you suggestion good and I'll do the modification. Thanks.